paper/paper.Rmd

```diff
@@ -48,25 +74,89 @@ knitr::opts_chunk$set(
+
+While other software toolkits exist, they often serve different purposes. The `{tidyverse}` [@Wickham2019], for example, provides a world-class environment for data manipulation and general-purpose plotting but does not focus on the intricacies of statistical model interpretation and reporting. Specialist packages like `{lme4}` [@bates_fitting_2015] for mixed-effects models or `{marginaleffects}` [@arel-bundock_how_2024] for marginal means are essential tools, but **easystats** serves as a complementary "meta-layer" that provides a single, easy-to-learn interface for interacting with the outputs from these and many other modeling packages. This allows researchers to focus on their scientific questions rather than on the technical idiosyncrasies of different software implementations.
+
+Ultimately, **easystats** fills a critical need in the R ecosystem for a user-friendly and cohesive suite of tools dedicated to the statistical modeling pipeline.
+
+# A Harmonized and Integrated Workflow
+
+A key design principle of the **easystats** ecosystem is the harmonization and integration of different packages into a simple, sequential workflow. The typical workflow for a statistical analysis using `{easystats}` starts with fitting a model, and then sequentially using different functions to obtain a comprehensive understanding of the model.
+
+It is worth noting that instead of loading each package separately, it is also possible to load all the core **easystats** packages in one go by loading the meta-package `{easystats}`.
+
+Let's demonstrate this with an example, where the user might start by preparing some data and fitting a simple linear model:
+
+```{r}
+library(easystats)
+data(coffee_data, package = "modelbased")
+
+# dichotomize outcome variable
+coffee_data$alertness <- categorize(coffee_data$alertness, lowest = 0)
+
+# rename variable
+coffee_data <- data_rename(coffee_data, select = c(treatment = "coffee"))
+
+# fit model
+model <- glm(
+  alertness ~ treatment,
+  data = coffee_data,
+  family = binomial()
+)
+```
+
+The `model` object can then be passed to functions from different **easystats** packages. For instance, the user can get a summary of the model parameters using the `{parameters}` package:
+
+```{r}
+model_parameters(model)
+```
+
+Then, the performance of the model can be assessed with the `{performance}` package:
+
+```{r}
+model_performance(
+  model,
+  metrics = c("AIC", "BIC", "R2", "PCP")
+)
```

Comment:
Would be great if this chunk could end up on the same page in the rendered output. Currently the header of the table is at the bottom of a page and the results are at the top of the following one.